### DISCUSSION TITLE
when to use mma and wmma instructions

### DISCUSSION THREAD

**Discussion by wzhcz8902** [NVIDIA/cutlass#1506] [Q&A] [answered]
mma and wmma instructions are both used to do mma computation, and there are several files about these two instructions under cutlass/arch, just eager to know which circumstances each of them is used.

**Comment by hwu36** [comment, marked as answer]
wmma is easier to use, but slower.  `mma.sync` ptx is hard to use, but faster.  it is hard to use because it takes some effort to prevent shared memory bank conflict.
